### DISCUSSION TITLE
Real Time Carbon Metrics from Cloud Providers

### DISCUSSION THREAD

**Discussion by adrianco** [Green-Software-Foundation/discussions#34] [Standards Working Group]
The project proposal for real time carbon metrics was made using a PRFAQ to define the outcome we would like to see, which is minute by minute metrics for energy usage and carbon footprint being shared by all the cloud providers using the same standard set of metrics. The proposal was approved on July 11th 2023 and it is now a supported project of the GSF Standards Working Group.

The PRFAQ and discussion are here:
https://github.com/Green-Software-Foundation/standards-wg/issues/67

This discussion has been setup to provide an open forum to talk about this standard as it develops.

**Comment by adrianco** [comment]
I'm currently gathering names of GSF members who'd like to be on the core team and meet to discuss and develop the standard.

**Reply by greenhsu123** [reply]
Count me in please!

**Reply by leonardpahlke** [reply]
I would be interested in supporting here!

**Reply by nikimanoledaki** [reply]
I would be interested too, but sadly I am not a member of the GSF since my org is not part of the foundation.

Still, the project is very relevant to the new [WG Green Reviews](https://github.com/cncf/tag-env-sustainability/pull/151) that we're setting up in the CNCF TAG Environmental Sustainability, which @leonardpahlke is involved with as well.

**Reply by adrianco** [reply]
I'm meeting on the Kepler community call on July 18th, will discuss then mechanisms to collaborate between CNCF and GSF. This will be developed in the open on GitHub, and I'm very interested in your input.

**Reply by leonardpahlke** [reply]
Sounds good. I'm very happy to hear that this project is open and inclusive 💯. ~I will see to participate in the Kepler call tomorrow as well~.

The Kepler call tomorrow is at 2am for me; will not be able to join.

**Reply by mrchrisadams** [reply]
Hi @adrianco - I'm really interested in this but I'm a little unsure of the time commitments for joining the core team.

Do you have a rough idea of how many calls in a given month it might be and how much time outside of them you might expect?

**Reply by sfontanarrosa** [reply]
Hi @adrianco, happy to join the initiative. I have been playing around with Kepler from time to time and seems really promising in addressing the granularity of the carbon emissions on the cloud environments, especially around K8 implementations.

Best,

**Reply by tmcclell** [reply]
I would love to join too please.

**Reply by PindyBhullar** [reply]
I am also interested in joining this initiative.

**Reply by adrianco** [reply]
Chris - I think we will have a weekly meeting, interested parties seem to span the Central Europe to US Pacific time zones at present, time outside that will be up to the individuals. Thanks for the shout out on the podcast!

**Reply by iivanoo** [reply]
@adrianco I would like to join as well, if possible.
We are running a lab at the Vrije Universiteit Amsterdam where we conduct experiments on the energy efficiency of software. We have years of experience on this and resources to carry out independent assessments of the metrics we will discuss. Our latest studies are available [here](https://s2group.cs.vu.nl/pages/greenlab/).

**Reply by raghupyapili** [reply]
@adrianco - Please loop me if its still open. We are also working on similar idea and would really like to understand it from an global perspective.

**Comment by Henry-WattTime** [comment]
This project now has it's own repo: https://github.com/Green-Software-Foundation/real-time-cloud
